spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/EnableModule.java

```diff
@@ -47,7 +46,7 @@
 @Configuration
 @MessageEndpoint
 @Import({RedisServiceConfiguration.class, RabbitServiceConfiguration.class,
-	ChannelBindingAdapterConfiguration.class, CodecConfiguration.class, ChannelBindingAdapterRunner.class,
```

Comment:
Yeah, I think we should strike a balance between explicit and auto configuration. Maybe use @EnableBinderAutoconfiguration/@EnableCodecAutoconfiguration as a hint (just a thought). I feel it very wrong that for adding Kafka support we have to expand @EnableModule again (and with the discussion of adding different bus implementations it becomes unwieldy)